starky/src/cross_table_lookup.rs

```diff
@@ -729,15 +740,23 @@ impl<'a, F: Field, const D: usize> CtlCheckVarsTarget<F, D> {
         // Get all cross-table lookup polynomial openings for each STARK proof.
         let ctl_zs = {
             let openings = &proof.openings;
-            let ctl_zs = openings.auxiliary_polys.iter().skip(num_lookup_columns);
+            let ctl_zs = openings
+                .auxiliary_polys
+                .as_ref()
+                .unwrap()
+                .iter()
+                .skip(num_lookup_columns);
             let ctl_zs_next = openings
                 .auxiliary_polys_next
+                .as_ref()
+                .unwrap()
                 .iter()
                 .skip(num_lookup_columns);
             ctl_zs.zip(ctl_zs_next).collect::<Vec<_>>()
         };
 
-        // Put each cross-table lookup polynomial into the correct table data: if a CTL polynomial is extracted from looking/looked table t, then we add it to the `CtlCheckVars` of table t.
+        // Put each cross-table lookup polynomial into the correct table data.
```

Comment:
"the correct table data" sounds odd. Is it "correct table's data" or "correct data table" or "data section of the correct table"?